joblib/test/test_numpy_pickle.py

```diff
@@ -245,17 +245,6 @@ def test_masked_array_persistence():
     nose.tools.assert_true(isinstance(b, np.ma.masked_array))
 
 
-def test_z_file():
```

Comment:
We should still keep the tests for the backward-compatibility layer.

I'd be in favour of moving it to numpy_pickle_compat.py or something and have test_numpy_pickle_compat.py.